### PR TITLE
Fixed allowing non-calvin email addresses

### PR DESCRIPTION
**Pull request by cdhazlett**
Added a condition to check that usernames end with @students.calvin.edu
or @calvin.

### CODE REVIEW

File: src/printapp/static/js/app.js [hunk 1]
```diff
@@ -89,7 +89,10 @@ App.SignInController = Ember.Controller.extend({
   actions: {
 
     signin: function() {
+
```
Comment (jglamine) [inline src/printapp/static/js/app.js]: Remove this blank line.

File: src/printapp/static/js/app.js [hunk 2]
```diff
@@ -99,6 +102,16 @@ App.SignInController = Ember.Controller.extend({
...
+      
+      if (!((username.indexOf('@students.calvin.edu') > -1) || (username.indexOf('@calvin.edu') > -1))) {
+          App.util.showAlert('#invalidEmailError');
+          controller.set('controllers.application.isAuthenticated', false);
+          controller.set('inputPassword', '');
+          controller.set('submitting', false);
+          return;
+      }
+
+
```
Comment (jglamine) [inline src/printapp/static/js/app.js:111]: Remove this blank line.

File: src/printapp/static/js/app.js [hunk 3]
```diff
@@ -99,6 +102,16 @@ App.SignInController = Ember.Controller.extend({
       }
 
       var controller = this;
+
```
Comment (jglamine) [inline src/printapp/static/js/app.js]: Remove this blank line.

File: src/printapp/templates/index.html [hunk 4]
```diff
@@ -29,6 +29,10 @@ <h3 class="subheader">Welcome</h3>
             The username or passphrase you entered is incorrect.
             <a href="#" class="close">&times;</a>
           </div>
+          <div id="invalidEmailError" data-alert hidden class="alert-box alert radius">
+            Please sign in with a Calvin Email address.
```
Comment (jglamine) [inline src/printapp/templates/index.html]: Change 'Email' to be lowercase.

**Comment by jglamine** [comment]
Looks good, see the comments I made for a few small formatting things.

**Comment by cdhazlett** [comment]
Updated the pull request.  Do more code review.

**Pull request merged**